.swiftpm/xcode/xcshareddata/xcschemes/swift-leeds.xcscheme

```diff
@@ -73,7 +73,8 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       launchStyle = "0"
-      useCustomWorkingDirectory = "NO"
+      useCustomWorkingDirectory = "YES"
+      customWorkingDirectory = "/Users/joewilliams/Documents/swiftleeds-web"
```

Comment:
Do we want to commit this?

Reply:
@adamrushy yeah, it's required so we can use the .env file. I've update the readme for this though.